test/commands/layout/push.test.ts

```diff
@@ -0,0 +1,313 @@
+import * as path from "node:path";
+
+import { expect, test } from "@oclif/test";
+import * as fs from "fs-extra";
+import { isEqual } from "lodash";
+import * as sinon from "sinon";
+
+import { factory } from "@/../test/support";
+import KnockApiV1 from "@/lib/api-v1";
+import { sandboxDir } from "@/lib/helpers/const";
+import { EmailLayoutData, LAYOUT_JSON } from "@/lib/marshal/email-layout";
+import { WithAnnotation } from "@/lib/marshal/shared/types";
+
+const layoutJsonFile = "default/layout.json";
+
+const mockEmailLayoutData: EmailLayoutData<WithAnnotation> = {
+  key: "default",
+  name: "Default",
+  html_layout:
+    "<!doctype html>\n" +
+    "<html>\n" +
+    " <body>\n" +
+    "  <p>This is some example text</p>\n" +
+    "  </body>\n" +
+    "</html>",
```

Comment:
I think here it might be easier to use the template literal syntax for a multi line string?

Example: https://github.com/knocklabs/knock-cli/blob/main/test/lib/marshal/conditions/helpers.test.ts#L16-L20